src/Console/Command/PurgeIssueFolderCommand.php

```diff
@@ -136,25 +142,19 @@ protected function findBranchesToCleanup(
         $remove = [];
         $notfound = [];
 
+        /* @var SplFileInfo $dir */
         foreach ($directories->directories() as $index => $dir) {
-            /* @var SplFileInfo $dir */
-            $issue = $this->fetchIssueUseCase->execute($dir->getFilename(), $branchNameFilter);
-
-            if ($issue) {
-                $issueStatus = strtolower($issue->fields->status->name);
-
-                $statusFound = in_array($issueStatus, $statuses, true);
-                if ($invert) {
-                    if (!$statusFound) {
-                        $remove[] = $dir;
-                    }
-                } elseif ($statusFound) {
-                    $remove[] = $dir;
-                }
-            } else {
+            $issue = null;
+            try {
+                $issue = $this->fetchIssueUseCase->execute($dir->getFilename(), $branchNameFilter);
+            } catch (JiraTicketNotFoundException $e) {
                 $notfound[] = $dir;
```

Comment:
```suggestion
                $remove[] = $dir;
```

I think the folder should be removed if the jira ticket was removed or never existed.
Or else we would end up with dead folders that would be scanned all over again.

Reply:
The folders in $notfound are being deleted in a later step in Line 120: `$this->removeNotFound($notfound, $io, $yes);`

Reply:
Ok, than its fine and we can leave it that way :)